Source/WebCore/css/parser/CSSPropertyParserWorkerSafe.cpp

```diff
@@ -207,6 +207,23 @@ RefPtr<CSSPrimitiveValue> CSSPropertyParserWorkerSafe::parseFontFaceDisplay(cons
     return parsedValue;
 }
 
+RefPtr<CSSValue> CSSPropertyParserWorkerSafe::parseFontFaceSizeAdjust(const String& string, ScriptExecutionContext& context)
+{
+    if (!context.settingsValues().cssFontFaceSizeAdjustEnabled)
+        return nullptr;
```

Comment:
I don't think you need this if you change the IDL as mentioned.

Reply:
Great to know, thanks!